### ISSUE TITLE
on changing orientation slide moves to 0th index instead of current index

### ISSUE THREAD

**Issue by KartikSagar007** [Veams/component-slider#7]
Hi,

/* change the active point in carousel /

While changing orientation of Mobile; here it is assigning my current slide to 0th index instead of current index.
Desired Output: If i change my view from portrait to landscape the slider should be same.

Please suggest on it.

**Comment by KartikSagar007** [comment]
In your code of "render" function if I change :
this.goToItem(this.startAtIndex + this.visibles) to this.goToItem(this.index)
it seems it is the solution using above code but now it changed my this.startAtIndex to last slider when I refresh the page. :(

Please suggest now.

**Comment by Sebastian-Fitzner** [comment]
Hi, I will take a look at that and post a solution in the next hours ...

**Comment by Sebastian-Fitzner** [comment]
So here we go, replace the `render()` method with that code snippet: 

``` js

render() {
	if (!Veams.currentMedia) {
		console.warn('Slider: Veams.currentMedia is necessary to support the slider module!');
		return;
	}

	if (this.$clonedLast && this.$clonedFirst) {
		this.$clonedLast.remove();
		this.$clonedFirst.remove();
		this.$items = this.$initialItems;
	}

	this.visibles = this.infinite ? 1 : this.options.visibleItems[ Veams.currentMedia ];
	this.itemsLength = this.$items.length;

	this.handleVisibility();

	if (!this.paginationDisabled) {
		this.removePagination();
		this.addPagination();
	}

	if (this.infinite) {
		this.infiniteLoop();
		this.index = this.index === 0 ? this.index + this.visibles : this.index;
	}

	this.bindTransitions();
	this.getAndSetDimensions();

	if (Veams.detections.touch && this.options.enableTouchSwipe && !this.touchSwipeEnabled) {
		this.bindSwipes();
	}

	this.goToItem(this.index);

	if (this.autoPlay && this.paused) {
		this.play();
	}
}
```

The important part is `this.index = this.index === 0 ? this.index + this.visibles : this.index;`. Now it should work like expected. Will publish that in the next release...

**Comment by Sebastian-Fitzner** [comment]
Is published in version 5.1.2

**Comment by KartikSagar007** [comment]
Thank you Sebastian :)
